contribute/linux/development_tree.rst

```diff
@@ -0,0 +1,161 @@
+.. _development_tree:
+
+Linux SOF drivers
+#################
+
+.. contents::
+   :local:
+   :depth: 3
+
+Background
+**********
+
+The Linux development is split by subsystem. All SOF contributions are
+merged through the sound/ system (maintained by Takashi Iwai) and
+sound/soc subsystem (maintained by Mark Brown).
+
+All SOF patches merged by the two maintainers will be used for
+linux-next (as a first pass of integration to detect conflicts with
+other subsystems or compilation issues) and eventually merged by Linus
+Torvalds in the mainline.
+
+
+Instructions for SOF developers
+*******************************
+
+Development branch
+==================
+
+All SOF development takes place on the topic/sof-dev branch in the SOF tree:
+
+git@github.com:thesofproject/linux.git
+
+Developers are required to submit Pull Requests (PRs) against the
+topic/sof-dev branch. The Continuous Integration (CI) will run a set
+of static analysis, builds and on-device testing.
+
+Two approvers are required for each PR. SOF admins may in some
+exceptions use their priviledges to merge PRs, e.g. to restore
+functionality and broken builds.
+
+When a PR is submitted by an SOF admin, it is required that another
+admin approves a PR. The PRs are integrated into the SOF tree using
+the 'rebase-and-merge' method which keeps the integrated patches in a
+linear order.
+
+Rebasing tree
+=============
+
+In addition to the topic/sof-dev branch, the SOF project maintains a
+parallel topic/sof-dev-rebase branch. This branch is not intended for
+development, but mostly to make upstream contributions easier to
+manage.
+
+Upstream merges
+===============
+
+During the Linux development, patches to the ALSA/ASoC cores,
+dependencies such as audio codecs or bug fixes may be contributed by
+the community. SOF Linux maintainers will on a regular basis,
+typically weekly, merge all upstream contributions into the SOF tree.
+
+
+Development flow
+****************
+
+The SOF Linux maintainers are
+
++---------------+-------------------+---------------+
+| Intel	        | Pierre Bossart    | @plbossart    |
++---------------+-------------------+---------------+
+| Intel         | Ranjani Sridharan | @ranj063      |
++---------------+-------------------+---------------+
+| Intel         | Kai Vehmanenaran  | @kv2019i      |
++---------------+-------------------+---------------+
+| NXP           | Daniel Baluta     | @dbaluta      |
++---------------+-------------------+---------------+
+
+The SOF maintainers apply the following process
+
+1. Mirror all SOF patches to topic/sof-dev-rebase
+*************************************************
+
+This mirroring consists in doing a set of "git cherry-pick" operations
```

Comment:
@plbossart typo here? "consists of"?

Reply:
https://english.stackexchange.com/questions/61600/consist-in-vs-consist-of

Reply:
It depends...

consists of a set of operations
consists in applying a set of cherry-pick operations

Reply:
Ahh thats a good lesson. I've never used "consists in" before. It just sounds weird to me. But then English is a funny language!

Reply:
@plbossart I have never used consist "in" in my entire life! :) Thanks for calling it to our attention. Out of curiosity, I looked it up. Still, after reading the explanation, I can see how we could get away with using "of" in your example. Here's what I read:

Consist of means to be composed or made up of, while 
consist in means to have the thing mentioned as the only or most important part.

Ex: The body consists of cells.
Her music consists in inspiring others.